docs/security/authorization/kibana-privileges.asciidoc

```diff
@@ -2,14 +2,30 @@
 [[kibana-privileges]]
 === Kibana privileges
 
-This section lists the Kibana privileges that you can assign to a role.
+Privileges have levels that you can use to manage which features users can access. Roles have privileges to determine whether users have write or read access.
```

Comment:
Instead of trying to discuss the concept of "levels", would it simplify this to just have two sections: "Base privileges" and "Feature privileges"?

Reply:
Something like this?
![image](https://user-images.githubusercontent.com/3493255/56295857-6a6b8580-60fb-11e9-9994-5477c17838fa.png)

Reply:
I think that's perfect, since we won't be referencing this from the "Role Management UI docs", do you mind integrating this within the role api docs themselves?

Reply:
I'll make those changes, but I'd prefer to wait on integrating until we have the rest of the doc updates in place: we have almost 10 references to this section today, so I want to see what those sections look like after we've updated them for Feature Controls.

Reply:
@kobelb Last we talked, we had discussed including more context here to help users understand where this is used. What are your thoughts on something like this?

![image](https://user-images.githubusercontent.com/3493255/56667862-90a1a000-667c-11e9-8540-60e672602d14.png)

Reply:
I like it!